landing_page/spec/views/microposts/show.html.erb_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'microposts/show.html.erb' do
+  before :each do
+    @user = FactoryBot.create(:user)
+    session[:user_id] = @user.id
+    login(@user)
+  end
+
+  context 'when the micropost has data' do
+    it 'displays a micropost correctly' do
+      assign(:micropost, FactoryBot.build(:micropost, user_id: @user.id))
+
+      render
+
+      expect(rendered).to have_content('Well hello there!')
+      expect(rendered).to_not have_content('Lockdown')
```

Comment:
What is the significance of 'Lockdown'?

Reply:
It's just testing that a the page doesn't contain something that's not specified within the post

Reply:
Ok you could test the page not to have 'coronavirus' as well? hopefully that passes..:)

But seriously, not sure there is value in a test that asserts that a page doesn't have a randomly selected string. The odds that the random string would be erroneous injected to your page are slim to none.

Reply:
Resolved in commit a62feca